source/agora/node/Ledger.d

```diff
@@ -125,9 +129,24 @@ public class Ledger
 
     ***************************************************************************/
 
-    public const(Block)* getLastBlock () @safe nothrow @nogc
+    public ref const(Block) getLastBlock () @safe nothrow
+    {
+        static Block block;
```

Comment:
This is a dangerous pattern: It means that when calling this function again, the previous reference will be invalidated.

Reply:
A block ref parameter was added.